ajax/libs/san/package.json

```diff
@@ -0,0 +1,20 @@
+{
+  "name": "san",
+  "filename": "san-router.min.js",
+  "description": "A Flexible JavaScript Component Framework",
+  "version": "3.6.0",
+  "homepage": "https://baidu.github.io/san/",
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/baidu/san.git"
+  },
+  "keywords": [
+    "San",
+    "Javascript",
+    "Framework",
+    "MVVM",
+    "Component",
+    "Web"
+  ],
+  "license": "MIT"
```

Comment:
Where is npm autoupdate config???
You should add npm autoupdate config with `**/!(README.md)*` pattern

Reply:
Even package.json need to be in 3.6.0? And the project san doesn't provide autoupdate config file, could I imitate autoupdate from another PR?

Reply:
Yes dist folder
and pattern I provide to you above.

Reply:
@sashberd I encounter "rebase with the latest master branch" problem, and when I want to rebase this branch with my latest master branch, it shows up a bunch of uncommit file which I even not modified before, could I commit those file and continue the rebase process?

Reply:
@LuckyPigeon I have no idea how to use application you are using. I suggest you to use command line if you want to learn git and contribute to cdnjs in the right way

Reply:
@sashberd I have done all the change of this PR, please review again, and I think I have already move all the files from dist to 3.6.0 except package.json because other PR put their package.json in the dist folder as well, so I'm not sure if I need to move my package.json to folder 3.6.0 or not. Thanks!